.storybook/meta/contribution.stories.mdx

```diff
@@ -93,3 +93,101 @@ $ npm run storybook
 See [Vocabulary guidelines].
 
 [Vocabulary guidelines]:https://cc-vocabulary.netlify.com/?path=/docs/vocabulary-contribution--page
+
+## Changing icons
+
+### Setting up requirements
+
+We use SVGs from the Font Awesome icon set for our icons. Get a copy from their
+[GitHub repository](https://github.com/FortAwesome/Font-Awesome) or from the
+[Download page](https://fontawesome.com/how-to-use/on-the-web/setup/hosting-font-awesome-yourself).
+
+To add, remove, or otherwise modify icons, you need to use [IcoMoon](https://icomoon.io/). IcoMoon is a custom
+icon-font builder, which allows selecting icons and converting them to fonts. IcoMoon can be used without an account,
+but it's recommended to have one.
+
+### Working in IcoMoon
+
+Open the IcoMoon app.
+<img src="guide/app_button.png" style={{ maxHeight: '48px' }} />
+
+If this is your first time, the Vocabulary project will not be loaded and, instead, one named 'Untitled Project' will.
+Spot this icon in the top right corner.
+<img src="guide/untitled_project.png" className="guide-image" style={{ maxHeight: '32px' }} />
```

Comment:
I think this screenshot could cover more of the screen. It's not immediately clear that this is a button on the app